src/Toggle/StatelessToggleChecker.php

```diff
@@ -3,12 +3,13 @@
 
 use Clearbooks\Labs\Client\Toggle\Entity\Group;
 use Clearbooks\Labs\Client\Toggle\Entity\User;
-use Clearbooks\Labs\Client\Toggle\Gateway\TogglePolicyGateway;
-use Clearbooks\Labs\Client\Toggle\UseCase\IsToggleActive;
+use Clearbooks\Labs\Client\Toggle\Gateway\GroupTogglePolicyGateway;
 use Clearbooks\Labs\Client\Toggle\Gateway\ToggleGateway;
+use Clearbooks\Labs\Client\Toggle\Gateway\TogglePolicyGateway;
+use Clearbooks\Labs\Client\Toggle\Gateway\UserTogglePolicyGateway;
 use Clearbooks\Labs\Client\Toggle\UseCase\Response\TogglePolicyResponse;
 
-class ToggleChecker implements IsToggleActive
+class StatelessToggleChecker implements UseCase\StatelessToggleCheckable
```

Comment:
Hm I think the interface name could just stay ToggleChecker or IsToggleActive as it was before. The interface can't enforce stateless behaviour, it's misleading to call it that in my opinion.